### PR TITLE
[DROOLS-954] fix removal of a rule with a sub-subnetwork

### PR DESCRIPTION
**Pull request by mariofusco**

### CODE REVIEW

File: drools-compiler/src/test/java/org/drools/compiler/integrationtests/IncrementalCompilationTest.java [hunk 1]
```diff
@@ -2043,4 +2047,38 @@ public void testSegmentMergeOnRuleRemovalWithNotExistingSegment() throws Excepti
...
+
+        KieServices ks = KieServices.Factory.get();
+
+        ReleaseId releaseId1 = ks.newReleaseId("org.kie", "test-upgrade", "1.1.1");
+        KieModule km = createAndDeployJar(ks, releaseId1, drl1);
+
+        KieContainer kc = ks.newKieContainer(km.getReleaseId());
+        KieSession ksession = kc.newKieSession();
+
+        kc.updateToVersion(releaseId1);
```
Comment (etirelli) [inline drools-compiler/src/test/java/org/drools/compiler/integrationtests/IncrementalCompilationTest.java:2071]: Out of curiosity, why do you call updateToVersion() using the same releaseId the current container was created with?

**Comment by etirelli** [comment]
+1 to merge

File: drools-compiler/src/test/java/org/drools/compiler/integrationtests/IncrementalCompilationTest.java (hunk 1 above)
Comment (mariofusco) [inline drools-compiler/src/test/java/org/drools/compiler/integrationtests/IncrementalCompilationTest.java:2071]: That's just a copy/paste mistake. I'll remove it, thanks.

**Comment by mariofusco** [comment]
Merged by https://github.com/droolsjbpm/drools/commit/1a6710b7f

**Pull request closed without merging**